app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
+    $('.label-warning.result-filter-tag').closest('tr').show();
+    $('.label-danger.result-filter-tag').closest('tr').show();
+    $('.label-success.result-filter-tag').closest('tr').show();
   }
   if(level == 'notice'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').hide();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
```

Comment:
For this `if-block`, you can use multiple selectors here.
i.e 
```
if(level == 'notice'){
  $('.label-info.result-filter-tag, .label-warning.result-filter-tag, .label-danger.result-filter-tag').closest('tr').show();
  $('.label-default.result-filter-tag, .label-success.result-filter-tag').closest('tr').hide();
}
```
In same way, you can apply for other if-blocks i.e for levels warning/error.